acquisitions/src/main/resources/thunderjet/mod-finance/features/ledger-fiscal-year-rollover-fail-resistance-when-duplicate-encumbrance.feature

```diff
@@ -0,0 +1,175 @@
+@parallel=false
+# for https://folio-org.atlassian.net/browse/MODFISTO-461
+Feature: Verify fault tolerance ledger fiscal year rollover when occurred duplicate encumbrance
+
+  Background:
+    * print karate.info.scenarioName
+
+    * url baseUrl
+
+    * callonce login testAdmin
+    * def okapitokenAdmin = okapitoken
+    * callonce login testUser
+    * def okapitokenUser = okapitoken
+
+    * def headersUser = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenUser)', 'Accept': 'application/json' }
+    * def headersAdmin = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitokenAdmin)', 'Accept': 'application/json' }
+
+    * configure headers = headersUser
+
+    * callonce variables
+
+    * def createFiscalYear = read('classpath:thunderjet/mod-finance/reusable/createFiscalYear.feature')
+    * def createLedger = read('classpath:thunderjet/mod-finance/reusable/createLedger.feature')
+    * def createOrder = read('classpath:thunderjet/mod-orders/reusable/create-order.feature')
+    * def createOrderLine = read('classpath:thunderjet/mod-orders/reusable/create-order-line.feature')
+    * def openOrder = read('classpath:thunderjet/mod-orders/reusable/open-order.feature')
+
+    * def fromFiscalYearId = callonce uuid1
+    * def toFiscalYearId = callonce uuid2
+    * def ledgerId = callonce uuid3
+    * def rolloverId = callonce uuid4
+    * def fundId = callonce uuid5
+    * def budgetId = callonce uuid6
+    * def orderId = callonce uuid7
+    * def poLineId = callonce uuid8
+
+    * def codePrefix = callonce random_string
+    * def fromYear = callonce getCurrentYear
+    * def toYear = parseInt(fromYear) + 1
+
+    * configure retry = { count: 10, interval: 5000 }
+
+  Scenario Outline: Prepare fiscal year with <fiscalYearId> for rollover
+    * configure headers = headersAdmin
+    * def fiscalYearId = <fiscalYearId>
+    * def code = <code>
+    * def periodStart = code + '-01-01T00:00:00Z'
+    * def periodEnd = code + '-12-30T23:59:59Z'
+
+    * def v = call createFiscalYear { id: #(fiscalYearId), code: #(codePrefix + code), periodStart: #(periodStart), periodEnd: #(periodEnd) }
+
+    Examples:
+      | fiscalYearId     | code     |
+      | fromFiscalYearId | fromYear |
+      | toFiscalYearId   | toYear   |
+
+
+  Scenario: Prepare finances
+    * configure headers = headersAdmin
+    * def v = call createLedger { id: #(ledgerId), fiscalYearId: #(fromFiscalYearId) }
+    * def v = call createFund { id: #(fundId),  ledgerId: #(ledgerId) }
+    * def v = call createBudget { id: #(budgetId), fundId: #(fundId), fiscalYearId: #(fromFiscalYearId), allocated: 100 }
+
+
+  Scenario: Create an order
+    * configure headers = headersAdmin
+    * def v = callonce createOrder { id: #(orderId), orderType: 'One-Time', reEncumber: true }
+
+
+  Scenario: Create an order line
```

Comment:
There should be only few scenarios in this file. Scenarios defined like this are organizational but skew reporting of test results and forces overuse of `callonce`. Please replace each scenario line with a comment.

CC: @SerhiiNosko

Reply:
Thunderjet has long used this structure in integration tests. If we start doing it more like we're supposed to with karate, we should stop using `@parallel=false` and make sure different scenarios can be executed in parallel. My latest [PR](https://github.com/folio-org/folio-integration-tests/pull/1310) can serve as an example.

Reply:
Please @okolawole-ebsco , @damien-git . Review it again. Thank you!

Reply:
@Abdulkhakimov Looks good!